suite-common/message-system/config/config.v1.json

```diff
@@ -1808,7 +1808,7 @@
                 },
                 "cta": {
                     "action": "external-link",
-                    "link": "https://trezor.io/learn/a/what-is-trezor-suite-lite#open-chat",
+                    "link": "https://trezor.io/guides/trezor-suite/getting-to-know-trezor-suite#open-chat",
```

Comment:
FYI: this link will be applied only with the new release of message system config, but I believe it's not important to do the release because of this change. We might rather consider removing this specific message...